packages/business-events/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@dvsa/business-events",
+  "version": "1.0.0",
+  "description": "A package to help manage business events in a domain context.",
+  "author": "DVSA",
+  "license": "ISC",
+  "files": [
+    "index.js",
+    "business-event.js",
+    "event-factory.js",
+    "event-publisher.js",
+    "business-event-validator.js",
+    "LICENSE",
+    "package.json"
+  ],
```

Comment:
I think we can make this slightly more dynamic if we do 

```
  "files": [
    "**.{js,ts}",
    "!jest.config.ts",
    "!src/__tests__/*.ts",
    "LICENSE",
    "package.json"
  ],
  ```

It will mean any new TS files added to the `src/` of this package, they will be automatically included